flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/nodes/exec/testutils/BatchRestoreTestBase.java

```diff
@@ -228,10 +228,10 @@ void loadAndRunCompiledPlan(TableTestProgram program, ExecNodeMetadata metadata)
 
         compiledPlan.execute().await();
         for (SinkTestStep sinkTestStep : program.getSetupSinkTestSteps()) {
-            List<String> expectedResults = getExpectedResults(sinkTestStep, sinkTestStep.name);
-            assertThat(expectedResults)
-                    .containsExactlyInAnyOrder(
-                            sinkTestStep.getExpectedAsStrings().toArray(new String[0]));
+            List<String> actualResults = getActualResults(sinkTestStep, sinkTestStep.name);
+            String[] expectResults =
+                    sinkTestStep.getExpectedBatchResultsAsStrings().toArray(new String[0]);
```

Comment:
I guess we don't need `.toArray(new String[0]);`
and could compare just lists
or did I miss something?

Reply:
I was just gently refactoring the existing code (rather than making changes).  

Switched to a list and using `containsAll`.